webapp/src/Components/Replay/ReplayViewer/ThreeScene.jsx

```diff
@@ -1,31 +1,47 @@
-import React, {Component} from 'react';
-import * as THREE from 'three';
+import React, {Component} from 'react'
+import * as THREE from 'three'
 
 export class ThreeScene extends Component {
+    constructor(props) {
+        super(props);
```

Comment:
This file is also weirdly linted - imports don't have `;`s while everything else does?

Is this a standard or is it just inconsistent?

Reply:
It's a *.jsx file so the linter had different rules for it and wanted semicolons for each statement (which I prefer). The linter didn't have a rule on the import statements though so I left them off.